covid19-etl/etl/cityofchicago.py

```diff
@@ -372,28 +378,29 @@ def files_to_submissions(self):
             f"Latest submitted date: {latest_submitted_date}. Getting data until date: {today}"
         )
 
-        self.summary_location_submitter_id = format_submitter_id(
+        summary_location_submitter_id = format_submitter_id(
             "summary_location",
             {"country": self.country, "state": self.state, "city": self.city},
         )
 
-        self.get_summary_location()
+        self.get_summary_location(summary_location_submitter_id)
         self.parse_cityofchicago_file(
             latest_submitted_date.strftime("%Y-%m-%dT%H:%M:%S"),
             today.strftime("%Y-%m-%dT%H:%M:%S"),
+            summary_location_submitter_id,
         )
         print("Done in {} secs".format(int(time.time() - start)))
 
     def submit_metadata(self):
         # Submits the data in `self.summary_locations`, `self.summary_clinicals` and `self.summary_group_demographic` to Sheepdog.
         print("Submitting data...")
-        if len(self.summary_locations) > 0:
-            print("Submitting summary_location data")
-            for sl in self.summary_locations.values():
-                sl_record = {"type": "summary_location"}
-                sl_record.update(sl)
-                self.metadata_helper.add_record_to_submit(sl_record)
-            self.metadata_helper.batch_submit_records()
+        print(self.summary_group_demographics)
```

Comment:
is this a debug statement that should be removed?

Reply:
Yes. Making a new commit with all above changes Incorporated.